build.gradle

```diff
@@ -29,13 +29,10 @@ intellij {
   if (project.hasProperty("ijVersion")) {
     version ijVersion
   } else {
-    version '2016.3.2'
+    version '2016.1.4'
```

Comment:
Changing the value from `2016.3.2` down to `2016.1.4`... Why that version is better, and how is it an appropriate choice?

Reply:
Well spotted - this is a mistake, I though I had changed it back.